ibis/backends/tests/test_array.py

```diff
@@ -36,7 +38,7 @@ def test_array_scalar(backend, con, alltypes, df):
 
 
 # Issues #2370
-@pytest.mark.notimpl(["impala", "datafusion"])
+@pytest.mark.notimpl(["impala", "datafusion", "duckdb"])
```

Comment:
I think most of these should be expressible, I'll see if I can push up the implementations to get them passing.